tests/unit/components/filters/Datetimefilter.spec.ts

```diff
@@ -43,4 +45,53 @@ describe('Datetimefilter.vue', () => {
     expect(vm.dateRange.startDate).toBe(null)
     expect(vm.dateRange.endDate).toBe(null)
   })
+
+  it('creates a date from a given string', () => {
+    const value = '1/1/2000'
+    // @ts-ignore
+    const date = vm.createDateFromValue(value)
+    expect(date).toEqual(new Date(Date.parse(value)))
+  })
+
+  it('creates a date from a given amount of milliseconds from epoch', () => {
+    const epochValue = 1588068623458
+    // @ts-ignore
+    const date = vm.createDateFromValue(epochValue)
+    expect(date).toEqual(new Date(epochValue))
+  })
+
+  it('creates a representable date string without the time component, if time is set to false', () => {
+    const noTimeProps = {
+      name: 'datetime',
+      label: 'Datetime',
+      collapsed: false,
+      time: false,
+      type: 'date-time-filter'
+    }
+    wrapper = mount(DateTimeFilter, { localVue, propsData: noTimeProps })
+
+    const dateToStringify = new Date(1588068623458)
+    const comparisonDate = dateToStringify.toLocaleDateString()
+    // @ts-ignore
+    const dateString = vm.dateString(dateToStringify)
+    expect(dateString).toEqual(comparisonDate)
+  })
+
+  it('creates a representable date with time string, when time is set to true', () => {
+    const dateTimeToStringify = new Date(1588068623458)
+    const comparisonDate = dateTimeToStringify.toLocaleString()
+    // @ts-ignore
+    const dateTimeToString = vm.dateTimeString(dateTimeToStringify)
+    expect(dateTimeToString).toEqual(comparisonDate)
+  })
```

Comment:
Deze test maakt het component meer rigide dan nodig is. Eigenlijk staat hier: Maak een Date aan (new Date(1588068623458) en vergelijk de uitkomst van 2x dezelfde date.toLocaleDateString. Als het gaat om puur coverage, zou je misschien ook die 2 helper functies naar `beautifyDateTime` kunnen halen en die doortesten.